compiler+runtime/test/jank/form/try/pass-multiple-catch-nested.jank

```diff
@@ -0,0 +1,23 @@
+;; Test nested try with multiple catches
+(assert (= :inner-caught
+           (try
+             (try
+               (throw :inner-exception)
+               (catch cpp/jank.runtime.object_ref e1
+                 :inner-caught)
+               (catch cpp/jank.runtime.object_ref e2
+                 :should-not-reach))
```

Comment:
This should not be possible. We have two catches of the same type. That is not permitted in C++ and we need to catch this during analysis.